src/ResourceManager/Sql/Commands.Sql/TransparentDataEncryption/Cmdlet/GetAzureRmSqlManagedInstanceKeyVaultKey.cs

```diff
@@ -29,9 +29,9 @@
 namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Cmdlet
 {
     /// <summary>
-    /// Defines the Get-AzureRmSqlManagedInstanceKeyVaultKey cmdlet
+    /// Defines the Get-AzureRmSqlInstanceKeyVaultKey cmdlet
     /// </summary>
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlManagedInstanceKeyVaultKey", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet)]
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceKeyVaultKey", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet)]
     [OutputType(typeof(AzureRmSqlManagedInstanceKeyVaultKeyModel))]
```

Comment:
Did you mean to leave the models with the old syntax?

Reply:
Yes, models were left intentionally with the old name